src/vllm_tgis_adapter/http.py

```diff
@@ -57,6 +31,7 @@ async def run_http_server(
 
     await serve_http(
         app,
+        engine,
```

Comment:
This has been added after 0.5.4, so we need a backwards compatible way to handle this until a new version is released

Reply:
ah F :(